controls/radmaskedinput/features/maskedinput-controls/currency.md

```diff
@@ -48,25 +50,111 @@ Here is a simple definition of a __RadMaskedCurrencyInput__ control:
 ![](images/radmaskedinput_currencyinput_default.png)
 
 You can further customize the __RadMaskedCurrencyInput__ control's behavior by setting the IsCurrencySymbolVisible property to False thus hiding the culture specific currency symbol. By default this property is set to True.
-				
-You can further restrict the user input by setting the __FormatString__ property.It uses [Standard Numeric Format Strings](http://msdn.microsoft.com/en-us/library/dwhawy9k.aspx) and [Custom Numeric Format Strings](http://msdn.microsoft.com/en-us/library/0c899ak8.aspx) to further format the __Text__ property.				
 
-#### __[XAML] Example 2: Setting the FormatString property__
-{{region xaml-radmaskedinput-features-controls-currency_1}}
+## Properties

```

Comment:
This is something that can be easily seen using F12. IMO we don't need sections listing properties, because they don't need much value. Instead, I think it is better to show each important property in a different section in the property, along with a code snippet and picture where applicable.